wire-ios/Wire-iOS/Wire-Info.plist

```diff
@@ -171,6 +171,29 @@
 	</array>
 	<key>UTExportedTypeDeclarations</key>
 	<array>
+		<dict>
+			<key>UTTypeConformsTo</key>
+			<array>
+				<string>public.data</string>
+			</array>
+			<key>UTTypeDescription</key>
+			<string>Wire Universal Backup File</string>
+			<key>UTTypeIconFiles</key>
+			<array/>
+			<key>UTTypeIdentifier</key>
+			<string>com.wire.backup-universal</string>
+			<key>UTTypeSize320IconFile</key>
+			<string>logo</string>
+			<key>UTTypeSize64IconFile</key>
+			<string>logo</string>
```

Comment:
This file doesn't exist, but I created a copy of the existing types.
Eventually we should discuss if we want to display an icon on these files and if the app should be opened by tapping a .wirebackup file.

Reply:
I've asked but no response yet.